src/subcommands/init

```diff
@@ -25,7 +25,7 @@ cmd-init() {
   echo "$PREFIX Selecting network interface"
   ETH_LINK=$(get-network-interface-name)
 
-  NODE_IP=$(ifconfig $ETH_LINK | grep -oP 'inet \K\S+')
+  NODE_IP=$(ip -o route show to default | awk '{print $3}')
```

Comment:
We need to find a cross-platform solution for getting the IP address of the host.
This command returns a wrong IP address when using DigitalOcean.